src/fsharp/NameResolution.fs

```diff
@@ -1352,6 +1360,9 @@ type TcResultsSinkImpl(g, ?source: string) =
     let capturedMethodGroupResolutions = ResizeArray<_>()
     let allowedRange (m:range) = not m.IsSynthetic       
 
+#if FABLE_COMPILER
```

Comment:
@dsyme This is a work-around for #676, unfortunately. There is more of it here and there. I'll change the conditional to be explicit about it.